draft-pda-protocol.md

```diff
@@ -114,6 +116,7 @@ document.
    input-validation protocol.
 1. Report: Uploaded to the leader from the client. A report contains the
    secret-shared and encrypted input and proof.
```

Comment:
Since we're getting into the -arity of things: does a report contain shares of a _single_ input or could several inputs be encoded into a single report?

Reply:
I think one input per report. As has been said, it might be that the "input" is actually comprised of multiple measurements. The "input" is meant to capture the thing that's validated in a single run of the input validation protocol.